onnxruntime/core/providers/cuda/nn/conv_transpose.cc

```diff
@@ -41,14 +41,15 @@ Status ConvTranspose<T>::DoConvTranspose(OpKernelContext* context, bool dynamic_
 
   const Tensor* X = context->Input<Tensor>(0);
   const TensorShape& x_shape = X->Shape();
-  const auto& x_dims = x_shape.GetDims();
+  auto x_dims = x_shape.GetDims();
   auto x_data = reinterpret_cast<const CudaT*>(X->template Data<T>());
 
-  if (X->Shape().NumDimensions() != 4) {
+  auto x_dimensions = X->Shape().NumDimensions();
+  if (x_dimensions != 4 && x_dimensions != 3) {
     // This condition is not true for two tests in ONNX tests series:
     // test_convtranspose_1d_cpu, test_convtranspose_3d_cpu.
```

Comment:
is the comment about test_convtranspose_1d still valid?

Reply:
Can you also uncomment 
https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/test/onnx/main.cc#L576
and see if the test passes now?

Reply:
Good idea!

Reply:
onnx test pass after remove the line.